tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Controllers/OrganisationControllerTests.cs

```diff
@@ -144,9 +144,9 @@ public static async Task GetAllAsync_ListOfOrganisationsExist_ReturnsTheOrganisa
 
             var organisationList = organisationResult.Organisations.ToList();
 
-            organisationList[0].Should().BeEquivalentTo(org1, config => config.Excluding(o => o.LastUpdated));
-            organisationList[1].Should().BeEquivalentTo(org2, config => config.Excluding(o => o.LastUpdated));
-            organisationList[2].Should().BeEquivalentTo(org3, config => config.Excluding(o => o.LastUpdated));
+            organisationList[0].Should().BeEquivalentTo(org1, config => config.Excluding(o => o.LastUpdated).Excluding(o => o.RelatedOrganisations).Excluding(o => o.ParentRelatedOrganisations));
+            organisationList[1].Should().BeEquivalentTo(org2, config => config.Excluding(o => o.LastUpdated).Excluding(o => o.RelatedOrganisations).Excluding(o => o.ParentRelatedOrganisations));
+            organisationList[2].Should().BeEquivalentTo(org3, config => config.Excluding(o => o.LastUpdated).Excluding(o => o.RelatedOrganisations).Excluding(o => o.ParentRelatedOrganisations));
```

Comment:
We don't currently have any formal style rules around line length but I think these are probably getting a bit on the long side. Recommend breaking like this:

```c#
organisationList[0].Should().BeEquivalentTo(
    org1,
    config => config.Excluding(o => o.LastUpdated).Excluding(o => o.RelatedOrganisations).Excluding(o => o.ParentRelatedOrganisations));

organisationList[1].Should().BeEquivalentTo(// ...
```

Reply:
well theres a couple of ways to skin this cat, i would suggest this:

```
            organisationList[0].Should().BeEquivalentTo(
                org1,
                config => config.Excluding(o => o.LastUpdated)
                                .Excluding(o => o.RelatedOrganisations)
                                .Excluding(o => o.ParentRelatedOrganisations));
```
thoughts?

Reply:
Your suggestion is alright but would prefer it with a slight tweak:

```c#
organisationList[0].Should().BeEquivalentTo(
    org1,
    config => config
        .Excluding(o => o.LastUpdated)
        .Excluding(o => o.RelatedOrganisations)
        .Excluding(o => o.ParentRelatedOrganisations));
```

I prefer this because it doesn't cause my eyes to scan as far to the right. A small, and subjective, thing but I find that a bit more readable – it seems more natural to me.